runtime/jcl/common/mgmtinit.c

```diff
@@ -632,8 +632,8 @@ gcEndEvent(J9JavaVM *vm, UDATA heapSize, UDATA heapUsed, UDATA *totals, UDATA *f
 			memoryPool->peakMax = memoryPool->postCollectionMaxSize;
 		}
 
-		/* if a heap collection threshold is set, check whether we are above or below */
-		if (0 < memoryPool->collectionUsageThreshold) {
+		/* if a memory pool collection threshold is set and the memory pool is manaaged by the collector, check whether we are above or below */
+		if (0 < memoryPool->collectionUsageThreshold && j9gc_is_managedpool_by_collector(vm, (UDATA)(gcID & J9VM_MANAGEMENT_GC_HEAP_ID_MASK), (UDATA)(memoryPool->id & J9VM_MANAGEMENT_POOL_HEAP_ID_MASK))) {
```

Comment:
Would you please add extra brackets around "0 < memoryPool->collectionUsageThreshold" ?

Reply:
done